src/zenml/zen_stores/schemas/component_schemas.py

```diff
@@ -93,23 +93,43 @@ def update(
         self.updated = datetime.utcnow()
         return self
 
-    def to_model(self) -> "ComponentResponseModel":
+    def to_model(
+        self, _block_recursion: bool = False
+    ) -> "ComponentResponseModel":
         """Creates a `ComponentModel` from an instance of a `StackSchema`.
 
+        Args:
+            _block_recursion: If other models should be recursively filled
```

Comment:
This docstring could be a little more clear, it isn't actually other models being recursively filled but it even prevents the returned `ComponentResponseModel` to have a user in one case and `None` in the other?

Reply:
removed now